airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.json

```diff
@@ -26,6 +26,14 @@
         "type": "boolean",
         "description": "Include data from deleted campaigns, ads, and adsets.",
         "default": "false"
+      },
```

Comment:
this is the first step, but this needs to be passed into the connector so it would actually use this value. 

Also, could you add a test using a mocked HTTP request to verify this works?